### PR TITLE
Plone5

### PR DESCRIPTION
**Pull request by busykoala**
TODO:
- [x] add icons to tinymce (in the ftw.book plugins)
- [x] remove upgrade step (i.e. propertiestool.xml...)
- [x] fix tests

Close https://github.com/4teamwork/izug.organisation/issues/1514

This is the branch https://github.com/4teamwork/ftw.book/pull/103 where the master was merged into. Therefore we can close https://github.com/4teamwork/ftw.book/pull/103 as soon as this one is going to be more stable.

image

### CODE REVIEW

File: ftw/book/profiles/default/cssregistry.xml [hunk 1]
```diff
@@ -2,42 +2,11 @@
 <object name="portal_css">
 
   <stylesheet
-      id="++resource++ftw.book-resources/ftwbook.css"
+      id="++resource++ftw.book-resources/tinymce/tinymce_minimal.css"
```
Comment (jone) [inline ftw/book/profiles/default/cssregistry.xml]: @busykoala is the CSS of this file also part of the SCSS file (duplication)?

I would have expected the the tinymce CSS just contains an extract of the relevant statements of the theming.scss file.
This way the tinymce CSS is only loaded within tinymce and woudl not need to be registered as CSS resource in the registry.

The other approach would be to also register the `tinymce_minimal.css` in the theming SCSS registry.

I would like to eliminate the need to register a CSS ressource completely.

File: setup.py [hunk 2]
```diff
@@ -21,6 +21,8 @@
 ]
 
 extras_require = {
+    'plone4': [
+        'Products.TinyMCE'],
```
Comment (jone) [inline setup.py]: Did you run into a problem without that?
Since TinyMCE is part of Plone 4 this should not be necessary.

Comment (busykoala) [reply setup.py]: I did not try without. It was there already, I just moved it into an extra, because it wouldn't work in plone5.

Comment (jone) [reply setup.py]: @busykoala ok then, let's just remove the extra

File: ftw/book/profiles/default/cssregistry.xml (hunk 1 above)
Comment (busykoala) [reply ftw/book/profiles/default/cssregistry.xml]: The css found in `tinymce_minimal.css` is in there only. But I indeed forgot some remains. These were removed in https://github.com/4teamwork/ftw.book/pull/112/commits/a45b574c2dbc97098ada1d1ed740be18beecf338

File: setup.py (hunk 2 above)
Comment (busykoala) [reply setup.py]: Removed in https://github.com/4teamwork/ftw.book/pull/112/commits/46207158d9c3401176719ac7608951489140381e

**Comment by busykoala** [comment]
❗️ I've just noticed that the tinymce plugins for keyword and footnote are not working in plone4. Please do not merge yet.

**Comment by busykoala** [comment]
@4teamwork/plone ready to review 🦐

File: test-plone-4.3.x.cfg [hunk 3]
```diff
@@ -4,6 +4,7 @@ extends =
     sources.cfg
 
 package-name = ftw.book
+test-egg = ftw.book [tests]
```
Comment (jone) [inline test-plone-4.3.x.cfg]:
```suggestion
```
✂️ That is not needed because this is the default.

**Pull request merged**